apps/bfd-server/dev/api-changelog.md

```diff
@@ -1,5 +1,25 @@
 # API Changelog
 
+## BFD-1383 Update V2 line item allowed charge amount mapping
+
+The charge amount for Carrier and DME have been updated from submitted to allowed.
```

Comment:
Replace with "Previously, the allowed charge amount in the EOB FHIR response was incorrectly being populated by the submitted charge amount data field. The value for allowed charge amount is now being populated correctly by the respective allowed charge amount data field. Note that this is only for v2"